app/screens/home/search/results/file_card.tsx

```diff
@@ -0,0 +1,139 @@
+            paddingVertical: 8,
+
+            borderWidth: 1,
+            borderRadius: 4,
+            borderColor: changeOpacity(theme.centerChannelColor, 0.16),
+
+            shadowColor: changeOpacity(theme.centerChannelColor, 0.16),
+            shadowOffset: {
+                width: 0,
+                height: 2,
+            },
+            shadowOpacity: 0.8,
+            elevation: 1,
+        },
+        flexRow: {flexDirection: 'row'},
+        iconContainer: {
+            marginHorizontal: 8,
+        },
+        textContainer: {
+            flex: 1,
+            flexDirection: 'column',
+            flexGrow: 1,
+        },
+        nameText: {
+            color: theme.centerChannelColor,
+            ...typography('Body', 200, 'SemiBold'),
+        },
+        infoText: {
+            color: changeOpacity(theme.centerChannelColor, 0.64),
+            ...typography('Body', 75, 'Regular'),
+        },
+        channelText: {
+            marginRight: 4,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+            borderRadius: 4,
+            paddingHorizontal: 4,
+        },
+        threeDotContainer: {
+            alignItems: 'flex-end',
+            marginHorizontal: 20,
+        },
+    };
+});
+
+export default function FileCard({fileInfo}: Props) {
+    const theme = useTheme();
+    const style = getStyleSheet(theme);
+    const size = getFormattedFileSize(fileInfo.size);
+
+    // TODO: hook this up
+    const openGallery = () => {
+        /* eslint-disable no-console */
+        console.log('Open Gallery');
+    };
+
+    // TODO: hook this up
+    const handleThreeDotPress = () => {
+        /* eslint-disable no-console */
+        console.log('pressed 3 dot');
+    };
+
+    // TODO: Get the channel name and display
+    const channelName = () => { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return 'channelName';
+    };
+
+    return (
+        <TouchableOpacity
+            onPress={openGallery}
+            style={style.container}
+        >
+            <View style={style.iconContainer}>
+                <FileIcon
+                    file={fileInfo}
+                    iconSize={48}
+                />
+            </View>
+            <View style={style.textContainer}>
+                <Text
+                    style={style.nameText}
+                    numberOfLines={1}
+                    ellipsizeMode={'tail'}
+                >
+                    {fileInfo.name}
+                </Text>
+                <View style={[style.flexRow]}>
+                    {/* <Text style={[style.infoText, style.channelText]}>{channelName()}</Text> */}
+                    <Text style={style.infoText}>{`${size} • `}</Text>
+                    <FormattedDate
+                        style={style.infoText}
+                        format={format}
+                        value={fileInfo.create_at as number}
+                    />
+                </View>
```

Comment:
when a long channel name is present this is going to cause an overflow..

here is a suggestion on how to handle that (needs clean up)

```
<View style={[style.flexRow]}>
                    <View style={{flexShrink: 1}}>
                        <Text
                            style={[style.infoText, style.channelText]}
                            numberOfLines={1}
                        >
                            {channelName()}
                        </Text>
                    </View>
                    <View style={{flexDirection: 'row', flexGrow: 1}}>
                        <Text style={style.infoText}>{`${size} • `}</Text>
                        <FormattedDate
                            style={style.infoText}
                            format={format}
                            value={fileInfo.create_at as number}
                        />
                    </View>
                </View>
```

Reply:
That worked nicely

<img width="500" alt="image" src="https://user-images.githubusercontent.com/7575921/174894110-fc0d95de-2a73-4119-adae-ac9a52364161.png">